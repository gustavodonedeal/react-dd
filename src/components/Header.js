import React from 'react';
import { Link } from 'react-router-dom';
import logo from '../donedeal-logo.svg';

class Header extends React.Component {
  render() {
    return (
      <header className="header-container">
        <div className="header-content">
          <div className="menu-bar-container">
            <div className="menu-bar-content page-row">
              <Link to={'/'} className="header-logo">
                <img src={logo} className="header-logo-image" alt="DoneDeal" />
              </Link>
            </div>
          </div>
        </div>
      </header>
    )
  }
}

export default Header;
