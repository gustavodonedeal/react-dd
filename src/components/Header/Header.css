:local .logoContainer {
  display: grid;
  justify-content: center;
  align-items: center;
}

:local .logo {
  width: 50vw;
}
