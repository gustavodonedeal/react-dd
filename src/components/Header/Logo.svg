<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 17.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="250px" height="43px" viewBox="0 0 250 43" enable-background="new 0 0 250 43" xml:space="preserve">
<g>
	<g>
		<path fill="#E42C27" d="M0.2,41.6V0h14.1C20,0,24.5,0.9,27.8,2.5s5.9,4.2,7.9,7.5c1.9,3.3,2.9,6.9,2.9,10.8c0,2.8-0.6,5.5-1.6,8
			c-1.1,2.5-2.6,4.8-4.6,6.7c-2,2-4.4,3.5-7.1,4.5c-1.6,0.6-3,1-4.3,1.2c-1.3,0.2-3.8,0.3-7.5,0.3H0.2z M13.7,5.3H6.2v31h7.7
			c3,0,5.3-0.2,7-0.6c1.6-0.4,3-0.9,4.1-1.5c1.1-0.6,2.1-1.4,3-2.3c2.9-3,4.4-6.8,4.4-11.3c0-4.5-1.5-8.1-4.5-11
			c-1.1-1-2.4-1.9-3.8-2.6c-1.5-0.7-2.8-1.1-4.1-1.3C18.6,5.4,16.5,5.3,13.7,5.3z"/>
		<path fill="#E42C27" d="M55.3,14.1c4.2,0,7.7,1.3,10.4,4c2.8,2.7,4.1,6.1,4.1,10.1c0,4-1.4,7.2-4.2,9.8c-2.8,2.6-6.3,3.9-10.6,3.9
			c-4.1,0-7.6-1.3-10.3-4c-2.8-2.7-4.1-6-4.1-9.9c0-4,1.4-7.4,4.2-10C47.6,15.5,51.1,14.1,55.3,14.1z M55,19c-2.6,0-4.7,0.9-6.4,2.6
			c-1.7,1.7-2.5,3.9-2.5,6.5c0,2.6,0.9,4.8,2.6,6.4s3.9,2.5,6.6,2.5s4.9-0.9,6.5-2.5c1.7-1.7,2.5-3.8,2.5-6.4s-0.9-4.8-2.7-6.5
			C59.9,19.9,57.7,19,55,19z"/>
		<path fill="#E42C27" d="M78.3,14.2v3.5c2.4-2.7,5.2-4,8.3-4c1.7,0,3.3,0.4,4.8,1.3c1.5,0.9,2.6,2.1,3.4,3.7c0.8,1.5,1.2,4,1.2,7.3
			v15.7h-5.4V26.1c0-2.8-0.4-4.8-1.3-6c-0.9-1.2-2.3-1.8-4.3-1.8c-2.6,0-4.8,1.3-6.5,3.9v19.6h-5.5V14.2H78.3z"/>
		<path fill="#E42C27" d="M123.7,27.9c0,0-13.3,1.8-19.3,0.8c-1.2-0.2,1.2,4.7,2.9,6.1c1.7,1.5,3.8,2.2,6.4,2.1
			c3.6-0.1,6.8-1.4,9.8-3.7l0.2,5.3c-1.6,1.2-3.3,2-4.9,2.6c-1.6,0.6-3.5,0.9-5.7,0.9c-3,0.1-5.5-0.4-7.4-1.6
			c-1.9-1.2-3.5-2.8-4.7-4.9c-1.2-2.1-1.8-4.5-2-7.3c-0.2-4.2,0.9-7.6,3.1-10.3c2.3-2.7,5.3-4.1,9-4.3c3.7-0.1,6.6,1,8.9,3.5
			c2.3,2.5,3.5,5.8,3.7,10.1L123.7,27.9z M105.6,25.5c0.8-0.6,12.9-0.9,13.8-0.6c0.9,0.3-0.9-3.8-2.1-4.9s-2.8-1.7-4.7-1.6
			c-1.9,0.1-3.5,0.7-4.7,2C106.6,21.5,104.8,26.1,105.6,25.5z"/>
		<path fill="#E42C27" d="M170.4,21.8c0,4.2-0.8,7.7-2.3,10.2c-1.5,2.6-3.4,4.6-5.5,6.2c-3,2.3-7.1,3.5-12.4,3.5h-0.6h-21.7V0.3
			h17.2c2.6,0,4.8,0.1,6.6,0.4c1.8,0.3,3.5,0.6,4.9,1.1c1.5,0.5,2.7,1,3.7,1.6c1,0.6,2,1.2,2.8,2c1.4,1.2,2.7,2.5,3.7,4.1
			c1.1,1.6,1.9,3.4,2.5,5.5C170.1,17.1,170.4,19.3,170.4,21.8z M157.7,21.2c0-3.4-0.6-6-1.9-7.9c-1.3-1.9-2.8-3.1-4.6-3.8
			c-1.8-0.6-3.6-0.9-5.5-0.9H140c2.1,3.5,1.6,21.6,0,24.5h5.7c3.3,0,6.1-0.8,8.5-2.3C156.5,29.2,157.7,26.1,157.7,21.2z"/>
		<path fill="#E42C27" d="M204.4,27.3c0,0-14.8,3.1-21.2,2c-1.4-0.3,1.6,3.3,3.3,4.3c1.7,1.1,3.6,1.5,5.6,1.4
			c3.6-0.2,7.6-1.4,11.8-3.7l0.3,6.6c-1.5,0.9-3.4,1.8-5.6,2.5c-2.2,0.8-5,1.3-8.5,1.5c-3.7,0.2-7-0.3-9.6-1.5
			c-2.7-1.2-4.8-3-6.2-5.3c-1.4-2.3-2.2-5-2.4-8c-0.2-3,0.3-5.7,1.5-8.1c1.2-2.4,3-4.4,5.6-5.8c2.5-1.5,5.7-2.3,9.4-2.5
			c2.5-0.1,4.9,0.3,7.2,1.2c2.4,0.9,4.3,2.6,5.9,4.9c1.6,2.3,2.5,5.4,2.8,9.2L204.4,27.3z M196.2,23.3c1.1,0.3-0.7-3.5-1.7-4.6
			c-1-1.1-2.5-1.6-4.3-1.5c-1.8,0.1-3.2,0.8-4.2,2c-1,1.2-2.6,5.2-1.4,4.7C186.1,23.4,194,22.8,196.2,23.3z"/>
		<path fill="#E42C27" d="M238.1,40.1c-1.6,0.6-3,1.1-4.3,1.5c-1.3,0.4-2.9,0.6-4.8,0.6c-1.8,0-3-0.3-3.7-0.9s-1.2-1.4-1.5-2.3
			c-1.2,0.9-2.5,1.7-3.8,2.3c-1.4,0.6-3.3,0.9-5.5,0.9c-2.2,0-4-0.4-5.3-1.2c-1.4-0.8-2.3-1.7-2.9-2.9s-0.8-2.4-0.8-3.6
			c0-1.8,0.5-3.4,1.4-4.6c0.9-1.2,2.2-2.2,3.8-3c1.6-0.8,3.7-1.6,6.3-2.4c1.5-0.5,2.6-0.8,3.4-1.1c0.8-0.3,1.3-0.4,1.7-0.6
			c0.4-0.1,0.8-0.3,1.2-0.5c0-1.5-0.3-2.6-0.9-3.4s-1.9-1.2-3.9-1.2c-2.4,0-4.5,0.5-6.5,1.5c-2,1-3.8,2.4-5.5,4v-8.3
			c1.3-0.9,2.8-1.7,4.3-2.4c1.5-0.6,3.2-1.1,4.9-1.4c1.7-0.3,3.4-0.4,5.1-0.4c4.4,0,7.7,0.9,9.9,2.5c2.2,1.7,3.3,4,3.3,7v11.7
			c0,1.1,0.1,1.9,0.3,2.5c0.2,0.6,0.6,0.8,1.2,0.8c0.7,0,1.6-0.3,2.8-0.9L238.1,40.1L238.1,40.1z M223.2,34.3v-7.1
			c-2.5,1-4.4,1.9-5.5,2.8c-1.2,0.8-1.8,1.9-1.8,3.4c0,1,0.3,1.8,0.9,2.3c0.6,0.5,1.2,0.8,2,0.8c0.7,0,1.5-0.2,2.2-0.6
			C221.7,35.3,222.5,34.9,223.2,34.3z"/>
		<path fill="#E42C27" d="M249.8,41.7H239V0.3h10.7V41.7z"/>
	</g>
</g>
</svg>
