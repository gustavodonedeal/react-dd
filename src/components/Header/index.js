import React from 'react';
import logo from './Logo.svg';

const Header = () => <img src={logo} />;

export default Header;
