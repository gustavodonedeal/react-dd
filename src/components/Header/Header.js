import React from 'react';
import { Link } from 'react-router-dom';
import logo from '../../assets/donedeal-logo.svg';
import './Header.css';

export default () => (
  <header className="header-container">
    <div className="header-content">
      <div className="menu-bar-container">
        <div className="menu-bar-content page-row">
          <Link to={'/'} className="header-logo">
            <img src={logo} className="header-logo-image" alt="DoneDeal" />
          </Link>
        </div>
      </div>
    </div>
  </header>
);
