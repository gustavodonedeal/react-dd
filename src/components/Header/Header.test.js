import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import Header from './Header';

it('renders correctly', async () => {
  const tree = renderer.create(
    <MemoryRouter>
      <Header />
    </MemoryRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
