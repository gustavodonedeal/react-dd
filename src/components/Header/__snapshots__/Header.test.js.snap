// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<div
  className="logoContainer"
>
  <img
    className="logo"
    src="test-file-stub"
  />
</div>
`;
